src/ru/help/guides/getting-started.md

```diff
@@ -0,0 +1,107 @@
+	
+	Альтернативно, вы можете скачать файл через нашу [страницу GitHub releases](https://github.com/FaizanDurrani/Paperback/releases/latest).
+1. Скачайте [AltServer](https://altstore.io/) для вашей оперативной системы.
+
+::: tip Пре-релизы
+ Если вы заинтересованны в функциях которые предоставлены в **Пре-релизах** или в **Стабильных** версиях, нужно быть [Patron](https://www.patreon.com/FaizanDurrani).
+:::
+
+::: warning Предупреждение
+ Для использования приложения требуется iOS 13.3+ или iPadOS 13.4+. Paperback НЕ БУДЕТ работать на версиях ниже.
+:::
+
+:::: el-tabs
+::: el-tab-pane label="macOS 10.14+"
+### AltStore
+#### Руководство
+1. Переместите "**AltServer.app**" в папку *Приложения* и запустите его.
+1. Подключите ваш **iPhone** или **iPad** на ваш компьютер и удостоверьтесь что он разблокирован и доверенный девайс.
+1. Пользователи **Mojave** должны открыть **iTunes** и включить "**iTunes Wi-Fi синхронизация**" для вашего телефона.
+   Пользователи **Catalina** должны открыть **Finder** и включить "**Показывать этот iPhone когда на WiFi**" для вашего телефона.
+1. Кликните иконку **AltServer** в меню статуса, нажмите "**Install AltStore**" и выберете свой телефон.
+1. Введите свой Apple ID имейл и пароль. (Адрес электронной почты и пароль чувствительны к регистру).
+	> *Ваш Apple ID и пароль посланны в толюко в Apple.*
+	- **AltServer** попросит вас установить плагин для имейла, инструкция ниже. Это требуется только в первый раз.
+2. После этого, AltStore будет установлен на вашем телефоне.
+ 
+ 
+### Имейл плагин AltStore
+1. Нажмите на "**Install Mail Plug-in**" в меню AltServer.
+1. Введите свой пароль что бы дать AltServer разрешение для установки плагина.
+1. Перезапустите Почту и откройте настройки `(CMD+,)`.
+1. Нажмите "**Управление Плагинами**" и включите "**AltPlugin.mailbundle**".
+1. Нажмите "**Применить и Перезапустить Почту**" что бы закончить установку.
+ 
+### Paperback
+#### Установка 
+1. Откройте **Почту** и **AltServer**.
+1. Удостовертесь что вы на _одном и том же интернет-соединение_ как компьютер на котором **AltServer**. Если вы не на одинаковом интернет-соединение, вы можете подключить ваш телефон к компьютеру.
+1. Доверьте приложение AltStore: на вашем телефоне зайдите в Настройки, Общие, Профили и Управление Устройствами. Выберите свой Apple ID и выберите **Доверить**.
+1. Запустите **AltStore** на вашем телефоне.
+1. Зайдите на **My Apps** и нажмите иконку **+** в верхнем левом угле.
+1. Откройте файл **`Paperback.IPA`** который был скачан из секции [Необходимые условия](/ru/help/guides/getting-started/#необходимые_условия).
+1. Наконецто, **Paperback** будет установленно!
+:::
+ 
+::: el-tab-pane label="Windows 10"
+### AltStore
+#### Руководство
+1. Скачайте и установите [iTunes](https://www.apple.com/itunes/download/win64) для Windows.
+   _Версия **iTunes** из Windows Store не подходить для этого процесса._
```

Comment:
You could use
```suggestion
   <el-tag type="warning">Not the Windows Store app!</el-tag>
```
It can be also used line 68 and then 71.

Reply:
Got it.